site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -255,12 +260,18 @@ endif::includeRoles[]
 [field]#application.samlv2Configuration.audience# [type]#[String]# [optional]#Optional# [default]#defaults to `issuer`# [since]#Available since 1.6.0#::
 The audience for the SAML response sent to back to the service provider from FusionAuth. Some service providers require different audience values than the `issuer` and this configuration option lets you change the `audience` in the response.
 
-[field]#application.samlv2Configuration.callbackURL# [type]#[String]# [required]#Required# [since]#Available since 1.6.0#::
+[field]#application.samlv2Configuration.authorizedRedirectURLs# [type]#[Array<String>]# [optional]#Optional# [since]#Available since 1.20.0#::
+A list of URLs that are authorized for FusionAuth to redirect the user after logging in via SAML.
+
+[field]#application.samlv2Configuration.callbackURL# [type]#[String]# [optional]#Optional# [since]#Available since 1.6.0# [deprecated]#Deprecated#::
```

Comment:
See `application.jwtConfiguration.algorithm` deprecation notice as an example. Once we mark deprecated, we should also add a note about when it was deprecated and what to use instead.